### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader: NVDA not announcing navigation key information at each node

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/azuredatastudio#20778]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG1.3.1; #FTP; #NVDA; #Win11; #JAWS

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that NVDA not announcing navigation key information at each node.

### Actual:
NVDA not announcing navigation key information at each node. 

**NOTE**: JAWS announcing as use arrow keys to navigate to expanded nodes, but it is not announcing specific key.

Expected Result:
NVDA should announce navigation key like "press right arrow key to navigate to next node".

Attachment:
![A11y_Azure Data Studio_Query Plan (Preview)_ScreenReader (4)](https://user-images.githubusercontent.com/97668775/194813866-8f21b170-c64a-4f47-9504-1e05999dc1ad.gif)

**Comment by aasimkhan30** [comment]
@poojapatil36, this issues if fixed in the latest insiders. Can you please verify and close the issue?

**Comment by Rama8345** [comment]
@aasimkhan30 
Verified the bug in the latest insiders build, but we are getting below error.
![image](https://user-images.githubusercontent.com/115216656/199224868-6facc11d-dc0c-4fb1-a672-6facd30d992c.png)

**Comment by poojapatil36** [comment]
Verified the issue in latest build
Issue is still reproducing, please find below attachment for reference.
![20778_Still repro](https://user-images.githubusercontent.com/97668775/199422326-8698a845-c0dd-4389-bb55-79cd2e7e706b.png)

**Comment by aasimkhan30** [comment]
@poojapatil36, based on my discussion with accessibility team during the office hours. We are announcing the navigation information only once when the user enter the graph.

Please check the highlighted line from your screenshot.
![image](https://user-images.githubusercontent.com/6816294/199823479-162db60a-99ce-4e20-9ad6-b3b5fa78d4a9.png)

**Comment by poojapatil36** [comment]
Closing this bug as it is reading the navigation key when we enter the node
